src/sentry/static/sentry/app/views/settings/project/projectTeams.jsx

```diff
@@ -85,17 +87,47 @@ const TeamRow = createReactClass({
 class ProjectTeams extends AsyncView {
   getEndpoints() {
     let {orgId, projectId} = this.props.params;
-    return [['teams', `/projects/${orgId}/${projectId}/teams/`]];
+    return [
+      ['projectTeams', `/projects/${orgId}/${projectId}/teams/`],
+      ['allTeams', `/organizations/${orgId}/teams/`],
+    ];
   }
 
   handleRemovedTeam(removedTeam) {
     this.setState({
-      teams: this.state.teams.filter(team => {
+      projectTeams: this.state.projectTeams.filter(team => {
```

Comment:
When updating state based on previous state, use a function as the first param. (This is because setState is async + batched)

e.g.
```
this.setState((prevState) => ({
       teams: prevState.teams.filter(team => {
...
```

https://reactjs.org/docs/react-component.html#setstate